src/test/resources/org/broadinstitute/hellbender/tools/copynumber/utils/combine-segment-breakpoints-no-samheader.tsv.seg

```diff
@@ -1,4 +1,5 @@
-#SAMPLE_NAME=SAMPLE1
+# This is another comment
+# This is yet another comment
```

Comment:
This test file is also not up to date.  The CNV-collections column headers are correct, but the SAM-style header is missing.  (I think you probably generated this file when the header consisted only of `#SAMPLE_NAME=...` and didn't update it when the CNV-collections format was updated?)

Reply:
As per discussions with @jonn-smith and @droazen , the AnnotatedInterval codec will support files that either contain comments (`#`) or a SAM File Header ('@').  Not both.  Then the locatable column headers is whatever is specified by the user config file or the default config file.  The default config file will match what the CNV collection classes use.

This test is of a file that is comments only and uses the default configuration file.  Therefore, the test should not have a SAM File Header and the locatable column headers should be CONTIG, START, and END.

Reply:
When would this file appear naturally in the wild?  My point is that it wouldn't (it would require that a user take a CNV-collections file, strip the SAM header, and add comments), so it's extremely confusing to base a test on it.  Test files should reflect the most typical use cases; if a file is atypical or unusual (e.g., an improperly or unexpectedly formatted file meant to stress test the code), then it should be clearly named as such.

Can you list here exhaustively the typical formats expected for each of the tools?  Ideally, I would be able to easily discern this just from glancing at the test files, but my main objection is that I cannot---precisely because most of them are a mishmash of conventions.

Reply:
Also, I think it might be more convenient to make the default config file specify the locatable column headers by indices 0-2.  For seg files that have `Sample` as the first column, I think it's a bit cleaner to just use `cut` externally than it is to change column headers with e.g. `sed`.